charts/aws-calico/templates/daemon-set.yaml

```diff
@@ -0,0 +1,156 @@
+kind: DaemonSet
+apiVersion: apps/v1
+metadata:
+  name: "{{ include "aws-calico.fullname" . }}-node"
+  labels:
+    app.kubernetes.io/name: "{{ include "aws-calico.fullname" . }}-node"
+{{ include "aws-calico.labels" . | indent 4 }}
+spec:
+  selector:
+    matchLabels:
+      app.kubernetes.io/name: "{{ include "aws-calico.fullname" . }}-node"
+  updateStrategy:
+    type: RollingUpdate
+    rollingUpdate:
+      maxUnavailable: 1
+  template:
+    metadata:
+      labels:
+        app.kubernetes.io/name: "{{ include "aws-calico.fullname" . }}-node"
+      {{- with .Values.calico.node.podAnnotations }}
+      annotations: {{- toYaml . | nindent 8 }}
+      {{- end }}
+    spec:
+      priorityClassName: system-node-critical
+      affinity:
+        nodeAffinity:
+          requiredDuringSchedulingIgnoredDuringExecution:
+            nodeSelectorTerms:
```

Comment:
does calico support ARM and windows? If not, you may want to set an OS and arch node selector.

Reply:
Yes, it does support ARM. I am working on this - https://github.com/aws/amazon-vpc-cni-k8s/issues/1218. Once I validate will update the image.